tutorials/test-nbvviewer/01-01-astronomical-CCD-image-components.ipynb

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">this is black code
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>